docs/content/direct/windows-setup-guide.md

```diff
@@ -0,0 +1,134 @@
+
+##  KubeStellar Installation Guide for Windows (PowerShell/CMD)
+
+## Work in Progress -- More steps coming soon!!!!!!
+
+### Prerequisites
+
+You need to install the following tools:
+
+1. [Git for Windows](https://git-scm.com/)
+2. [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+3. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/)
+4. [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+5. [Helm](https://helm.sh/docs/intro/install/)
+6. [OCM CLI (`clusteradm`)](https://open-cluster-management.io/)
+7. [kubeflex CLI (`kflex`)](https://github.com/kubestellar/kubeflex)
+
+
+
+###  Step-by-Step Windows Installation
+
```

Comment:
Too much bold here.
<img width="701" alt="image" src="https://github.com/user-attachments/assets/b3bb266c-374a-4a50-894c-f05b9d24b3d0" />

Also, as a reader, I was fairly lost. The first thing I see here is nothing that I was promised. This bold content is acctually a parenthetic remarks teaching somebody who does not already know about a utility built into Windows. It is currently marked as 4 sections that are peers of "Step-by-Step Windows Installation", but that is not accurate. Please use more appropriate markdown.